ga4gh/frontend.py

```diff
@@ -194,6 +194,7 @@ def configure(configFile=None, baseConfig="ProductionConfig",
     TODO Document this critical function! What does it do? What does
     it assume?
     """
+    requests.packages.urllib3.disable_warnings()
```

Comment:
Oooo, not sure about this. The user should most definitely be warned if they're using dodgy certs. Blanket suppressing this warning is a definite security no-no. Can we just do this during tests? Isn't there a Flask thing we can check to see if we're in testing mode? I guess this is tricky here because we're actually running the server in a subprocess, not running it in test mode. We could add an option to do this in the server CLI I suppose.